data-donation-sdk/src/commonMain/kotlin/care/data4life/datadonation/donation/fhir/FhirResourceBlurMapper.kt

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright (c) 2021 D4L data4life gGmbH / All rights reserved.
+ *
+ * D4L owns all legal rights, title and interest in and to the Software Development Kit ("SDK"),
+ * including any intellectual property rights that subsist in the SDK.
+ *
+ * The SDK and its documentation may be accessed and used for viewing/review purposes only.
+ * Any usage of the SDK for other purposes, including usage for the development of
+ * applications/third-party applications shall require the conclusion of a license agreement
+ * between you and D4L.
+ *
+ * If you are interested in licensing the SDK for your own applications/third-party
+ * applications and/or if you’d like to contribute to the development of the SDK, please
+ * contact D4L by email to help@data4life.care.
+ */
+
+package care.data4life.datadonation.donation.fhir
+
+import care.data4life.datadonation.donation.fhir.FhirContract.FhirResourceBlurMapper.Companion.REFERENCE_SEPARATOR
+import care.data4life.datadonation.donation.program.model.ProgramFhirResourceBlur
+import care.data4life.datadonation.donation.program.model.ProgramFhirResourceConfiguration
+
+internal object FhirResourceBlurMapper : FhirContract.FhirResourceBlurMapper {
```

Comment:
can't find any place where this object is used?

Reply:
We need to wire it up, once we know how we distributing incoming FHIR batches from a consumer. Basically I added it already since I had the feeling it is better to do it right away before I forget 50% of it.

Reply:
maybe move it to another pr instead of merging dead code?

Reply:
Feels a bit weird to me to provide an Entity without a mapper to be honest.

Reply:
Also - currently the only method to know how the mapping looks like actually, is by looking into the JS version of this. This at least spares me from headaches.

Reply:
which is still solved by another pr anyways